packages/math-input/src/components/keypad.tsx

```diff
@@ -96,9 +93,13 @@ class Keypad extends React.Component {
             return {
                 ...rest,
                 initialBounds: {
+                    // @ts-expect-error TS2533
```

Comment:
The component should have these values before rendering, but TS still thought they could be null/undefined.

Reply:
🤔  I'm not arguing for a change at the moment, but I think the first render it wouldn't have a DOM node (which is what `this._container` depends on) because we haven't rendered anything yet.  But I'm good with supressing these for now as this obvious is working in production right now.